index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio-setup-mobile-first</title>
+  <link rel="stylesheet" href="styles.css">
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="navigation">
+      <a href="https://github.com/TaiwoEnoch" class="logo">Enoch</a>
+      <img src="/images/Union.png">
```

Comment:
- [ ] [OPTIONAL] Please check if the hamburger is correctly displayed. I think there is some issue with the path. Kindly add a `.` before the path to fix it. 😎️

```
<img src="./images/Union.png">
```

Reply:
It is perfectly working if it is not the hamburger will have not display on the screen at all, Thanks for your observation.